src/dbwrapper.cpp

```diff
@@ -155,11 +156,25 @@ CDBWrapper::~CDBWrapper()
 
 bool CDBWrapper::WriteBatch(CDBBatch& batch, bool fSync)
 {
+    double mem_before = DynamicMemoryUsage() / 1024 / 1024;
```

Comment:
We might want to skip this call (and the one after) if BCLog::LEVELDB is disabled, with the string parsing and floating point arithmetic it seems a lot of overhead for a log message that will be ignored anyway.

Reply:
Good observation, fixed.